apps/xero/exceptions.py

```
@@ -267,7 +267,7 @@ def new_fn(*args):
             if not payment:
                 post_accounting_export_summary(expense_group.workspace.fyle_org_id, expense_group.workspace_id, [expense.id for expense in expense_group.expenses.all()], expense_group.fund_source, True)
 
-            if not payment and args[-1] == True:
+            if not payment and args[-2] == True:
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if args[-2]:` for truth checks

Replace with `args[-2]`

(E712)